btree/btree.c

```diff
@@ -2012,6 +2085,12 @@ static void generic_move(struct nd *src, struct nd *tgt,
 			 enum dir dir, int nr, struct m0_be_tx *tx);
 static bool ff_invariant(const struct nd *node);
 static bool ff_verify(const struct nd *node);
+static void ff_opaque_set(const struct segaddr *addr, void *opaque);
+static void *ff_opaque_get(const struct segaddr *addr);
+uint32_t ff_ntype_get(const struct segaddr *addr);
+/* uint16_t ff_ksize_get(const struct segaddr *addr); */
+/* uint16_t ff_valsize_get(const struct segaddr *addr);  */
```

Comment:
ff_ksize_get and ff_valsize_get are just similar to ff_keysize and ff_valsize respectively, Please check if you can use those functions instead of defining new function which returns same thing.

Reply:
I do not remember why I have taken these functions, must be thinking of some scenario. I have commented this code for now as it is not required. I will be cleaning this code if not required in future. 
ff_keysize/ff_valsize are taking node descriptor as parameter and I wanted it from segaddr. Yash will be taking up this activity to distinguish functions based on node descriptor and segaddr.

Reply:
okay.